doc/en/developer/source/programming-guide/app-schema/index.rst

```diff
@@ -160,3 +160,7 @@ For Oracle:
       For example, I installed the package using the System user. Then I gave onlinetestuser permission to execute it.
       I run the tests with -DSC4OUser=System so it knows to use the System.SC4O.ST_GeomFromEWKT method.
 
+Running MongoDB Online Tests 
+----------------------------
+
+MongoDB online tests expected by default that a MongoDB instance can be reached using host `127.0.0.1` and port `27017`. A different host can be provided using environment variable `mongo.host` and a different port can be provided using variable `mongo.port`. If no MongoDB instance can be reached the online tests will be skipped.
```

Comment:
As noted elsewhere, it would be preferable that these setting were stored in a ```mongodb.properties``` file.